portal-impl/src/com/liferay/portal/service/impl/ResourceBlockLocalServiceImpl.java

```diff
@@ -1107,4 +1109,4 @@ protected void updatePermissionsHash(ResourceBlock resourceBlock) {
 	private static final Log _log = LogFactoryUtil.getLog(
 		ResourceBlockLocalServiceImpl.class);
 
-}
\ No newline at end of file
+}
```

Comment:
@mario83 by looking the code this is not correct fix as you are removing resource block. Y